its/pom.xml

```diff
@@ -20,22 +20,14 @@
   </organization>
 
   <properties>
-    <scannerMsbuild.version>5.15.0.80890</scannerMsbuild.version>
     <surefire.argLine>-server</surefire.argLine>
   </properties>
 
   <dependencies>
     <dependency>
       <groupId>org.sonarsource.orchestrator</groupId>
       <artifactId>sonar-orchestrator-junit5</artifactId>
-      <version>4.4.0.1640</version>
-    </dependency>
-    <dependency>
-      <groupId>org.sonarsource.scanner.msbuild</groupId>
-      <artifactId>sonar-scanner-msbuild</artifactId>
-      <version>${scannerMsbuild.version}</version>
-      <type>zip</type>
-      <classifier>net46</classifier>
```

Comment:
@costin-zaharia-sonarsource 
Educational: was this dependency there to ensure that the zip was locally available for the orchestrator to use it? Or it was unnecessary, since the orchestrator was retrieving the zip of the scanner by itself anyway?